cmake/px4_add_library.cmake

```diff
@@ -44,7 +44,11 @@ function(px4_add_library target)
 	target_compile_definitions(${target} PRIVATE MODULE_NAME="${target}")
 
 	# all PX4 libraries have access to parameters and uORB
-	add_dependencies(${target} uorb_headers parameters)
+	# TODO: Exclusion of qurt is temporary until these modules
+	#       build cleanly.
+	if(NOT ${PLATFORM} MATCHES "qurt")
+		add_dependencies(${target} uorb_headers parameters)
```

Comment:
We might be able to remove this entirely.

Reply:
Should I remove it now or are you going to do some more research?

Reply:
Actually, uorb_headers does not need to be excluded. But parameters does. It opens a whole list of issues that will need to be dealt with when we get to the parameters module.